src/styles.css

```diff
@@ -3,3 +3,7 @@ body {
   color: white;
   background-color: #202020;
 }
+
+.openseadragon-container {
+	min-height: 500px;
+}
```

Comment:
There's probably a better, more 'responsive' way to do this, but I just added it to get it displaying